### PR TITLE
ADCM-1194 Fixed rotation log from file system

### PR DESCRIPTION
**Pull request by a-alferov**

### CODE REVIEW

File: python/cm/job.py [hunk 1]
```diff
@@ -953,14 +953,14 @@ def log_rotation():
...
     if log_rotation_on_fs:
-        rotation_jobs_on_fs = JobLog.objects.filter(
-            finish_date__lt=timezone.now() - timedelta(days=log_rotation_on_fs)).values('id')
-
-        if rotation_jobs_on_fs:
-
-            for job in rotation_jobs_on_fs:
-                shutil.rmtree(os.path.join(config.RUN_DIR, str(job['id'])))
-            log.info('rotation log from fs')
+        for folder in os.listdir(config.RUN_DIR):
```
Comment (acmnu) [inline python/cm/job.py:956]: Do you understand that it could be not only folders in RUN_DIR?

File: python/cm/job.py [hunk 2]
```diff
@@ -953,14 +953,14 @@ def log_rotation():
...
+            if not folder.startswith('.'):
```
Comment (acmnu) [inline python/cm/job.py:957]: What this code for? It is not obvious why we have hidden files in RUN_DIR. Please made a comment about.

Comment (a-alferov) [reply python/cm/job.py:957]: In the RUN_DIR folder is .gitignore

File: python/cm/job.py (hunk 1 above)
Comment (a-alferov) [reply python/cm/job.py:956]: And what else could be?

Comment (acmnu) [reply python/cm/job.py:956]: File, regular file. You change logic, and create a lot of dirs in run, but on old installation there are a lot of files there.

File: python/cm/job.py (hunk 2 above)
Comment (acmnu) [reply python/cm/job.py:957]: It should not be there. Supposed we have to fix this in dockerignore file. Please remove that code and add exception to dockerignore

Comment (a-alferov) [reply python/cm/job.py:957]: Ok

File: python/cm/job.py (hunk 1 above)
Comment (a-alferov) [reply python/cm/job.py:956]: If I understand correctly, you want me to add the deletion of files from the old log storage location LOG_DIR?

File: python/cm/job.py (hunk 2 above)
Comment (a-alferov) [reply python/cm/job.py:957]: code `if not folder.startswith('.'):` is used for development

File: python/cm/job.py (hunk 1 above)
Comment (a-alferov) [reply python/cm/job.py]: Added deletion of both folders and files

**Pull request merged**